### ISSUE TITLE
heatmap intensity not reflecting correctly

### ISSUE THREAD

**Issue by mapapo1** [jupyter-widgets/ipyleaflet#674]
Say I have a list of 5 locations with coordinates, and each location has a number I want to show on a heatmap, so I can visualize which loation has most 'hits'. For example

    Location 1: x,y,80
    
    Location 2: x,y,1
    
    Location 3: x,y,20
    
    Location 4: x,y,40
    
    Location 5: x,y,30

I’m creating the heatmap by adding the location to the heatmap with the intensity set to the number of times the count is, so 
```
heatmapLocations = []
for location in locations
                heatmapLocations.append((x,y,count))
heatmap = Heatmap(locations=heatmapLocations)
```
My heatmap is showing, and it shows the highest count as a red circle, so that’s correct, also location 3 with count 20 is shown as light yellow, which is also correct, but location 4 and 5 are shown the same as location 1, especially when I zoom in, as if they have the same density, although there value is half of location 1. How can I reflect this? I tried the different heatmap parameters as max_zoom, max, radius, etc, but can’t seem to get it right. I also tried to normalise my data by dividing the numbers by 100, so the values are 0.8, 0.2, etc. but with no effect. I think I'm missing something understanding the way the heatmap works.

**Comment by mapapo1** [comment]
By the way I'm using ipyleaflet version 0.13.3

**Comment by davidbrochart** [comment]
It also has to do with the number of points, as it does clustering, e.g.:
```python
from ipyleaflet import Map, Heatmap

latlngs = [[0, 0], [0, 1], [1, 1], [1, 0], [0.5, 0.5]]
intensities = [80, 1, 20, 40, 30]
locations = []
for ll, v in zip(latlngs, intensities):
    for _ in range(v):
        locations.append((ll[0], ll[1], v))
heatmap = Heatmap(locations=locations, radius=100, blur=15, max=1, maxZoom=8)

m = Map(center=[0.5, 0.5], zoom=8)
m.add_layer(heatmap)
m
```
But I agree it's not clear for me either. Do you want to raise an issue in https://github.com/mourner/simpleheat ?

**Comment by mapapo1** [comment]
I now raised the same issue at https://github.com/mourner/simpleheat, although I doubt I'll get an answer because the last activity there seems to be three years ago. 

Maybe my use case is a little bit different than usual for this kind of heatmap. I have a couple locations with hits on a location, so actually I don't have clustering (say 15 data points in and around New York and 7 in and around Boston) but I want to reflect the amount of 'hits' on one location, say 15 in the center of NY and 7 in the center of Boston.

**Comment by davidbrochart** [comment]
Have you tried the code I posted?

**Comment by mapapo1** [comment]
Yes, but I think I don't get what you mean. For each location, you are adding this location 5 times to the cluster?
Basically, I want to determine the intensity based on the occurrences on 1 location. 

With your code, I still have 3 locations showing the same intensity (red), I guess the ones with 100, 40 and 30 intensity.

**Comment by davidbrochart** [comment]
Yes you're right, I guess we should get more information in the issue you posted in simpleheat.
